CHANGELOG.md

```diff
@@ -402,7 +403,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Allow option to skip device selection page in demo app.
 - Add demo hook for debugging media connections
-- Add github link to getNearestMediaRegion method in README.md
+- [Demo] Fix serverless deploy script to not print out logs
```

Comment:
Request you to check once the existing master changelog. "[Demo] Fix serverless deploy script to not print out logs" is under **Fixed** section under `1.22.0` release, so keep it as is. You have removed it from `1.22.0` and added under 1.21.0 **Added** section.

1. Add "[Documentation] Link to Media Regions documentation in README" under **Fixed** section under [Unreleased]. (You have done this)
2. Keep "[Demo] Fix serverless deploy script to not print out logs" under **Fixed** section under `1.22.0` as currently seen in the master changelog.

There should only be 1 change in changelog and thats the addition of "[Documentation] Link to Media Regions documentation in README" under **Fixed** section under [Unreleased]

Reply:
ah sorry, got a little mixed up with my git remote configs -- will fix now